BUILD.md

```diff
@@ -86,7 +86,7 @@ For other system requirements and other dependencies, please see [this section](
 |**Build Shared Library**|--build_shared_lib||
 |**Build Python wheel**|--build_wheel||
 |**Build C# and C packages**|--build_csharp||
-|**Build WindowsML**|--use_winml --use_dml||
+|**Build WindowsML**|--use_winml --use_dml --build_shared_lib|WindowsML depends on DirectML and the OnnxRuntime shared library.|
```

Comment:
|.....|--user_winml \ --use_dml \ --build_shared_lib|.....|.
put --use_dml and --build_shared_lib in new lines might look better.

Reply:
ah, I was wondering how to do that! thanks!